README.md

```diff
@@ -1512,6 +1512,53 @@ would enable the use of a `@fold` on the `adjacent_animal` vertex field of `Foo`
 }
 ```
 
+## Schema Generation
+To generate a GraphQL schema, we first map a database schema into an intermediate representation:
+the `SchemaGraph`.
+
+#### SchemaGraph
+* `SchemaGraph`
+    * Keeps track of a schema's transitive inheritance structure.
+    * Keeps track of a schema's indexes. Indexes are not used in the GraphQL schema generation and 
+      have a provisional API.
+    * Represents a database's schema through `SchemaElement` objects.
+        * `SchemaElement`
+            * Can be one of three kinds:
+                * `VertexType`, which represent schema vertices.
```

Comment:
Our nomenclature here is somewhat inconsistent:
- three `kinds`
- vertex`type`
- nongraph`element`

Let's pick one and stick with it.